### ISSUE TITLE
Added Signature in Settings but not displaying on e-mails

### ISSUE THREAD

**Issue by crazybadger** [nextcloud/mail#2608]
I'm trying to use Nextcloud Mail for a small charity I'm working with, and have added a signature to the account which displays their registered number and company number.  It seems to be saving OK, as when I go back in it's still there, but it doesn't appear when I draft a new message, or on any of the test messages I've sent.  

I'm running Nextcloud 18.0.0 on a FreeNAS 11.2-U7 machine inside a FreeBSD 11.3 jail.

Thanks

**Comment by crazybadger** [comment]
Sorry, ignore that!  I've worked out the problem.  I'd set-up more than one e-mail account and the signature block will only work on the first one.  I moved the account to the 'up' to the top and it's now working.  Maybe this can be changed to a request for individual signatures for multiple mail accounts?  Thanks

**Comment by ChristophWurst** [comment]
Yep, we could totally have like a list of defined signatures and allow the user to assign them. Then you can share the same signature for multiple accounts.

**Comment by dumblob** [comment]
Having a list of predefined signatures would be great - we use it due to multilingual conversations mostly.

**Comment by pvrhartmann** [comment]
We came across the same issue. Setting up only one mailbox in the mail app results in delayed sending and the signatured would't be added unless you add a second mailbox and choose the mailbox you want to send from from the drop down menu - even if it's the first one.

Although there's a workaround the solution cannot be to be forced to set up another mailbox. This works totally fine for users who're using multiple mailboxes anyways but not for our average user.

We're currently using Nextcloud 22.2.8 with Mail 1.12.0 .

Is there any update on this issue? Do you need more information?

Best regards

**Comment by ChristophWurst** [comment]
@pvrhartmann your issue is tracked in https://github.com/nextcloud/mail/issues/6361.
